apis/types/image_info.go

```diff
@@ -34,6 +34,7 @@ type ImageInfo struct {
 	Os string `json:"Os,omitempty"`
 
 	// repository with digest.
+
```

Comment:
Please remove this line of addition. Since this file is automatically generated by swagger. We do not modify this.